lib/libc/Kconfig

```diff
@@ -221,7 +221,7 @@ endif # MINIMAL_LIBC
 
 config STDOUT_CONSOLE
 	bool "Send stdout to console"
-	depends on CONSOLE_HAS_DRIVER
+	depends on CONSOLE_HAS_DRIVER || X86_EFI_CONSOLE
```

Comment:
Seems like it would be better to select `CONSOLE_HAS_DRIVER` when `X86_EFI_CONSOLE` is chosen.

Reply:
Yes, you are right. And I found the problem, we don't need this anymore. Thanks for pointing this out!